tests/python/frontend/tflite/test_forward.py

```diff
@@ -2480,6 +2481,16 @@ def __test_elemwise(in_data):
                 inq0_min, inq0_max = (out_min, out_max)
                 inq1_min, inq1_max = (out_min, out_max)
 
+            if exclude_zero_point:
+                if inq1_max == inq1_min:
+                    raise ZeroDivisionError("Input range is 0.")
+
+                # only compute for rhs.
+                quant_scale = 255 / (inq1_max - inq1_min)
+                zero_point = int(round(-inq1_min * quant_scale))
+                data[1][data[1] == zero_point] += 1
+                data[1][data[1] == 0] += 1
```

Comment:
Is this second adjustment required?  Looking at the two next two each other, its seems like the first (when `data[1] == zero_point`) would only be required when these are pre-quantized values, and the second (when `data[1] == 0`) would only be required when these are post-quantization values, and that only one of those can be true at a time.

Reply:
I agree--but when I was testing, I needed to keep the second adjustment for all tests to pass.

Reply:
Interesting, and thank you for checking.  Possibly due to tests that quantize one or the other argument, leading `data` to be used in both contexts.